src/app/resources/resources.component.ts

```diff
@@ -109,7 +110,9 @@ export class ResourcesComponent implements OnInit, AfterViewInit, OnDestroy {
       switchMap((resources) => this.parent ? this.couchService.localComparison(this.dbName, resources) : of(resources))
     ).subscribe((resources) => {
       this.resources.data = resources.filter(
-        (resource: any) => this.excludeIds.indexOf(resource._id) === -1 && resource.doc.private !== true
+        (resource: any) => this.excludeIds.indexOf(resource._id) === -1
+        && resource.doc.private !== true || this.route.snapshot.data.myPersonals === true
```

Comment:
This condition isn't working.  It should be if the route is 'myPersonals' then we check `privateFor` otherwise `private !== true`.

Reply:
ok paul thank you.